src/test/java/no/entur/antu/validation/validator/xpath/rules/ValidateAuthorityRefTest.java

```diff
@@ -0,0 +1,79 @@
+package no.entur.antu.validation.validator.xpath.rules;
+
+import static no.entur.antu.validation.validator.xpath.rules.ValidateAuthorityRef.INVALID_AUTHORITY_REF_RULE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import no.entur.antu.validation.validator.organisation.OrganisationAliasRepository;
+import org.entur.netex.validation.test.xpath.support.TestValidationContextBuilder;
+import org.entur.netex.validation.validator.ValidationIssue;
+import org.entur.netex.validation.validator.xpath.XPathRuleValidationContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ValidateAuthorityRefTest {
+
+  private ValidateAuthorityRef validator;
+  private OrganisationAliasRepository organisationAliasRepository;
+
+  private final String EXISTING_ORGANISATION_ALIAS = "OrganisationAlias:1";
+  private final String NON_EXISTENT_ORGANISATION_ALIAS = "OrganisationAlias:2";
+
+  @BeforeEach
+  public void setUp() {
+    this.organisationAliasRepository = mock(OrganisationAliasRepository.class);
```

Comment:
You could use a simple implementation of this interface instead of a mock. Ideally mocks should be used only on classes we do not have control over.

Reply:
Agree. I have updated it to use the new implementation of `SimpleOrganisationAliasRepository` instead.